src/main/java/usecases/GameTemplate.java

```diff
@@ -58,6 +58,18 @@ public static GameTemplate GameFactory(String name, List<String> usernames, User
         return new CrazyEights(usernames, userManager, input, output);
     }
 
+    public static int GetMaxPlayers(String name) {
```

Comment:
Method name doesn't follow convention. I know `GameFactory` is also named incorrectly, PR #79 fixes that.